LINAC.c

```diff
@@ -0,0 +1,33 @@
+#include <xc.h>
+#include "LINAC.h"
+
+ void LINAC_init() {
+    TRISB1 = 0;     //set MOTOR_FWD as output
+    LATB1 = 0;
+    
+    TRISB2 = 0;     //set MOTOR_BWD as output
+    LATB2 = 0;
+    
+    TRISB3 = 1;     //set LIM_HIGH as input
+    ANSELB3 = 0;    //set as digital input
+    
+    TRISB4 = 1;     //set LIM_LOW as input
+    ANSELB4 = 0;    //set as digital input
+    
+}
+ 
+int open_vent(){
```

Comment:
Why does this return int? Should be void unless the return value means something (also, conventionally an "everything is normal" int return value is 0).